### PR TITLE
Added cog for random scary movie

### PR DESCRIPTION
**Pull request by rgables**
For issue #9: Random scary movie generator.

Currently picks a random "Halloween" movie.  We still need to figure out some kind of list of scary movies to randomly select from.

### CODE REVIEW

**Comment by jodth07** [comment]
I am not sure if [omdb](http://www.omdbapi.com/) have by genre in their api.
[OMDB](https://www.themoviedb.org/documentation/api?language=en-US) seems to though

**Comment by heavysaturn** [comment]
I think that for the purposes of this hacktoberfest, retrieving a random halloween movie will be good enough.

File: bot/cogs/movie.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+        await ctx.send(movie_details)
+
+    @staticmethod
+    async def select_movie():
+        """
+        Selects a random movie and returns a json of movie details from omdb
+        """
+
+        # TODO: Come up w/ a scary movie list to select from. Currently returns random Halloween movie
```
Comment (heavysaturn) [inline bot/cogs/movie.py]: This line can be removed - a random halloween movie will be fine.

File: bot/cogs/movie.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        omdb_params = {
+            'apikey': OMDB_API_KEY,
+            'i': selection
+        }
+        response = requests.get('http://www.omdbapi.com/', omdb_params)
+
+        return response.json()
+
+    @staticmethod
+    async def format_metadata(movie):
```
Comment (heavysaturn) [inline bot/cogs/movie.py]: I'd like to see this method return an embed instead.

On our PyDis bot, we have a snake movie command, which builds an embed like this with the data it gets from OMDB:

image

Would you be able to do something like that?

Comment (rgables) [reply bot/cogs/movie.py]: Yeah, that sounds do-able.  Thanks for the feedback.

**Comment by rgables** [comment]
> I am not sure if [omdb](http://www.omdbapi.com/) have by genre in their api.
> [OMDB](https://www.themoviedb.org/documentation/api?language=en-US) seems to though

I wasn't aware of themoviedb, but their api is way fancier.  Thanks for the tip

**Comment by rgables** [comment]
Switched from omdb to tmdb's api for the advanced search filters.  !movie also now embeds the movie poster + info instead of just linking to an outside website.
screen shot 2018-10-09 at 1 05 32 am

**Comment by heavysaturn** [comment]
That looks good!

**Pull request merged**